packages/bun-types/crypto.d.ts

```diff
@@ -3913,6 +3913,13 @@ declare module "crypto" {
     candidate: LargeNumberLike,
     options?: CheckPrimeOptions,
   ): boolean;
+  /**
+   * An implementation of the Web Crypto API standard.
+   *
+   * See the {@link https://nodejs.org/docs/latest/api/webcrypto.html Web Crypto API documentation} for details.
+   * @since v15.0.0
```

Comment:
The version flag is pointing to a nodejs version. In my opinion this should be omitted as it will lead to confusion.

Reply:
You are right, but this "node:xx" is "Node.js compatibility". The link here points to "Node", so maybe there's nothing wrong with that.
Of course, if it were you, you'd do this

--------
Or are you referring to "@since v15.0.0" as redundant?

Reply:
see latest commit. 🥳